Microsoft.VisualStudio.CodeTools/PropertyPageUI/PropertyPageUI.rc

```diff
@@ -44,7 +46,29 @@ BEGIN
     "#define VER_FILEDESCRIPTION_STR ""PropertyPage Package Resources\\0""\r\n"
     "#define VER_INTERNALNAME_STR ""PropertyPageUI.dll\\0""\r\n"
     "#define VER_ORIGINALFILENAME_STR ""PropertyPageUI.dll\\0""\r\n"
-    "#include ""EnvSdkVersion.ver""\r\n"
+    "VS_VERSION_INFO VERSIONINFO\r\n"
```

Comment:
:memo: The other option is creating the **EnvSdkVersion.ver** file with these contents.